spec/spec_helper_acceptance_local.rb

```diff
@@ -46,8 +46,8 @@ def install_dependencies
     if $facts['os']['family'] in ['SLES', 'SUSE'] {
       exec { 'Enable legacy repos':
         path    => '/bin:/usr/bin/:/sbin:/usr/sbin',
-        command => 'SUSEConnect --product sle-module-legacy/15.5/x86_64',
-        unless  => 'SUSEConnect --status-text | grep sle-module-legacy/15.5/x86_64',
+        command => 'SUSEConnect --product sle-module-legacy/15.6/x86_64',
```

Comment:
Can you use facts here...

Reply:
something like this
[https://github.com/puppetlabs/puppetlabs-java/pull/601](https://github.com/puppetlabs/puppetlabs-java/pull/601)

Reply:
Done @malikparvez.